micronaut-langchain4j-jlama/build.gradle.kts

```diff
@@ -16,5 +16,5 @@ tasks {
     }
 }
 micronautBuild {
-    binaryCompatibility.enabledAfter("1.0.1")
+    binaryCompatibility.enabled = false
```

Comment:
Why did you remove this? Either this should be `enabled = true` or `enabledAfter("a version which makes sense")`

Reply:
because micronaut-langchain4j-jlama has not been released. No idea why and why have already released 1.1.0